migration-engine/migration-engine-tests/tests/create_migration/create_migration_tests.rs

```diff
@@ -0,0 +1,250 @@
+use crate::*;
```

Comment:
only the last test is new, the rest was moved from `mod.rs` to this file with a more identifiable name